runner/app/live/streamer/process.py

```diff
@@ -24,10 +24,10 @@ def __init__(self, pipeline_name: str):
         self.ctx = mp.get_context("spawn")
 
         self.input_queue = self.ctx.Queue(maxsize=5)
-        self.last_input_time = 0.0
         self.output_queue = self.ctx.Queue()
-        self.last_output_time = 0.0
         self.param_update_queue = self.ctx.Queue()
+        self.error_queue = self.ctx.Queue()
+        self.log_queue = self.ctx.Queue(maxsize=100)  # Keep last 100 log lines
```

Comment:
Just so I understand, will this populate log_queue with an array of the last 100 log lines? @ecmulli just want your opinion on this, is this something we need in the event and with so much data?

Reply:
We keep 100 logs here but actually only send the last 10 lines in the event. I think at some point we should be streaming all the logs tho, that's what fal.ai does too. It will be useful for any lower-level developer access that we give to write pipelines (there are some comfy nodes to print stuff for example, useful for debugging).

Also, as I mentioned, restarts are way more common than errors, and on the restart case the logs are the only thing we can provide. So they are very useful IMO and a must-keep.